src/main/java/at/hannibal2/skyhanni/features/inventory/craft/CraftableItemList.kt

```diff
@@ -0,0 +1,129 @@
+package at.hannibal2.skyhanni.features.inventory.craft
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.SackAPI.getAmountInSacks
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.InventoryCloseEvent
+import at.hannibal2.skyhanni.events.InventoryOpenEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.CollectionUtils.addOrPut
+import at.hannibal2.skyhanni.utils.CollectionUtils.sortedDesc
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.itemName
+import at.hannibal2.skyhanni.utils.ItemUtils.itemNameWithoutColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.NEUItems
+import at.hannibal2.skyhanni.utils.NEUItems.getPrice
+import at.hannibal2.skyhanni.utils.NEUItems.isVanillaItem
+import at.hannibal2.skyhanni.utils.PrimitiveItemStack.Companion.toPrimitiveStackOrNull
+import at.hannibal2.skyhanni.utils.RenderUtils.renderRenderables
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.renderables.Renderable
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.recipes.CraftingRecipe
+import io.github.moulberry.notenoughupdates.recipes.NeuRecipe
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.math.floor
+
+class CraftableItemList {
+    private val config get() = SkyHanniMod.feature.inventory.craftableItemList
+
+    private var display = listOf<Renderable>()
+    private var inInventory = false
+    private val craftItemPattern by RepoPattern.pattern(
+        "craftableitemlist.craftitem",
+        "Craft Item"
+    )
+
+    @SubscribeEvent
+    fun onInventoryOpen(event: InventoryOpenEvent) {
+        if (!isEnabled()) return
+        if (!craftItemPattern.matches(event.inventoryName)) return
+        inInventory = true
+
+        val pricePer = mutableMapOf<NEUInternalName, Double>()
+        val lines = mutableMapOf<NEUInternalName, Renderable>()
+        val avaliableMaterial = readItems()
+        for (internalName in NEUItems.allInternalNames) {
+            if (config.excludeVanillaItems && !internalName.isVanillaItem()) continue
```

Comment:
```suggestion
            if (config.excludeVanillaItems && internalName.isVanillaItem()) continue
```
Either this is a small logic mistake from you or I don't understand what this setting does

Reply:
i changed include vanilla items to exclude vanilla items and have not properly changed the logic there. thanks for the catch!